Utils/release_notes_generator.py

```diff
@@ -20,8 +20,9 @@
 
 EMPTY_LINES_REGEX = re.compile(r'\s*-\s*\n')
 IGNORED_LINES_REGEX = re.compile(r'<!-[\W\w]*?-->')
-ENTITY_TYPE_SECTION_REGEX = re.compile(r'^#### (\w+)$\n([\w\W]*?)(?=^#### )|^#### (\w+)$\n([\w\W]*)', re.M)
-ENTITY_SECTION_REGEX = re.compile(r'^##### (.+)$\n([\w\W]*?)(?=^##### )|^##### (.+)$\n([\w\W]*)', re.M)
+ENTITY_TYPE_SECTION_REGEX = re.compile(r'^#### ([\w ]+)$\n([\w\W]*?)(?=^#### )|^#### ([\w ]+)$\n([\w\W]*)', re.M)
+ENTITY_SECTION_REGEX = re.compile(r'^##### (.+)$\n([\w\W]*?)(?=^##### )|^##### (.+)$\n([\w\W]*)|'
+                                  r'^- \*\*(.+)\*\*$\n([\w\W]*)', re.M)
```

Comment:
What does this need to catch? `r'^- \*\*(.+)\*\*$\n([\w\W]*)'`

Reply:
to catch the content entity instance name that is wrapped with `**` like here:
```
#### Incident Fields
- **XDR Alerts**
    - Second
```